### PR TITLE
Abdul/gpio one wire

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for GPIOoneWire app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : GpioOneWire.vcxproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.
    PlatformToolSet form v141 to v142

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

File: GPIOOneWire/CPP/GpioOneWire.vcxproj [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{5a9de0e6-1fa8-4f34-ab39-7473288becfa}</ProjectGuid>
+    <RootNamespace>GpioOneWire</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformVersion>10.0.19041.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
```
Comment (neilsh-msft) [inline GPIOOneWire/CPP/GpioOneWire.vcxproj:11]: In the readme, it says that it's using GpioChangeReader which is new to 19041, so shouldn't the min version be changed to 19041 to accommodate this?

File: GPIOOneWire/CPP/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+enabled. `GpioPin.SetDriveMode()` currently takes
+much longer when interrupts are enabled than when they are
+disabled. If you try to change a pin from output to input when interrupts
+are enabled, you will miss the response from the DHT22.
+As a workaround, two pins are used to interface with the DHT22.
+One pin is configured as an output and is used to assert the data line LOW
+to request a sample. The other pin is configured as input and is used to
+capture changes on the data line.
+
+Your breadboard should be wired as follows. You can use either a MOSFET or
```
Comment (neilsh-msft) [inline GPIOOneWire/CPP/ReadMe.md]: We probably want to have part numbers here, people are going to ask what BJT or MOSFET they should use.

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

**Review by rsameser — APPROVED** [review]

File: GPIOOneWire/CPP/ReadMe.md (hunk 2 above)
Comment (rsameser) [reply GPIOOneWire/CPP/ReadMe.md]: Agreed - please add @v-abdrahman

Comment (v-abdrahman) [reply GPIOOneWire/CPP/ReadMe.md]: Added the name of BJT and MOSFET in the readmes

File: GPIOOneWire/CPP/GpioOneWire.vcxproj (hunk 1 above)
Comment (v-abdrahman) [reply GPIOOneWire/CPP/GpioOneWire.vcxproj:11]: I have changed the MinimumPlatform to 15063 in readme, As we are using it consistently in all the samples.

**Pull request merged**